README.md

```diff
@@ -1,63 +1,200 @@
+
+![Dirty Dancing UI](https://s3.amazonaws.com/learn-verified/dirty-dancing-ui-blank.png)
+
+It would be nice to have a title in that space, wouldn't it? Remember, one of the obvious benefits (although perhaps not the _most_ important) of a navigation controller is that it allows you to easily add a title to your UI.
+
+Changing the title displayed in the top navigation bar is not as obvious as it seems, and understanding how to do so requires a bit of understand of how a navigation controller works. Remember how a navigation controller sits "above" the other view controllers? The navigation controller is responsible for managing a stack of views. The current view (be it a table view or some other view) is at the top of the stack. As the user navigations within your app, views are popped off the top of the stack (if the user goes "back") or push on top (if the user goes "forward").
+
+The title displayed in the navigation bar is not the title of the navigation controller; rather, the navigation controller gets that title from the view on the top of the view stack, and draws that in the navigation bar.
+
+This is a wordy way of saying that the title is actually set in the topmost view controller—in this case, the table view.
+
+In the object listing, you should see an item under **Table View Controller Scene** called **Navigation Item**.
+
+![Navigation Item](https://s3.amazonaws.com/learn-verified/navigation-item.png)
+
+Click once on that item to select it. Then, in its Attributes Inspector, change its title to "Dirty Dancing". You should see that title appear in the navigation bar.
+
+![Navigation Bar](https://s3.amazonaws.com/learn-verified/navigation-item-full.png)
+
+Build and run your app again. It still won't do anything cool, but you'll see the title in the nav bar now.
+
+![Dirty Dancing UI](https://s3.amazonaws.com/learn-verified/navigation-bar-title.png)
+
+## Table View Controllers
+
+Let's write some code to make your app actually _work_. Switch back to the `TableViewController.swift` file. This file currently contains one class, `TableViewController`. As the name suggests, this is a table view controller—specifically, the one that will power your app! Let's get to work on it.
+
+First, you're going to need a way to store the names of the songs that make up the _Dirty Dancing_ soundtrack. It makes sense to store them in a property, doesn't it? And since it's a list of songs, an `Array` of `String`s (`[String]`) seems like an appropriate data type. Create a property called `dirtyDancingSoundtrack` to store these items. The songs that comprise the _Dirty Dancing_ soundtrack are:
+
+1. "(I've Had) The Time of My Life"
+2. "Be My Baby"
+3. "She's Like the Wind"
+4. "Hungry Eyes"
+5. "STAY"
+6. "Yes"
+7. "You Don't Own Me"
+8. "Hey Baby"
+9. "Overload"
+10. "Love Is Strange"
+11. "Where Are You Tonight?"
+12. "In the Still of the Night"
+
```

Comment:
It's funny to think.. when an idea hits me... something as silly as.. Ok it would be cool to talk about the Dirty Dancing soundtrack to see it be seriously implemented in discussing something like TableView Controllers. Amazing.